bin/fresh-node10

```diff
@@ -23,6 +23,9 @@ for arg in "$@"; do
 		# Option: -env
 		#   Forward "MW_*" and "MEDIAWIKI_*" environment variables from the host machine
 		#   to the Fresh container. This is designed for use with MediaWiki browser tests.
+		#   If a .env file exists in the directory, its MW_* and MEDIAWIKI_* environment
+		#   variables will be forwarded to the container; and any values set at
+		#   runtime take precedence over environment variables set in the .env file.
```

Comment:
@Krinkle should the `.env` override locally exported variables, or keep it how it is now?

Reply:
Good question! I can see it both ways.

If you mainly let tools like mediawiki-docker manage the wiki, then it would be nice to have it default to the current directory's `.env`. Assuming that that file would be generated correctly by those tools and thus naturally be the "right' one.

Letting the values from `.bashrc` win in that case would be undesirable as in this situation those would only be intended as a default for when invoking Fresh from a directory that doesn't have `.env`.

On the other hand, if I have something like MediaWiki-Docker-Dev or Vagrant, which supports having multiple wikis, then the natural way to switch wikis is to set different environment variables. So letting the `.env` win there would make that rather frustrating I imagine, in that you'd be running `export MW_FOO=1` and then invoke `fresh -env` and yet the variable didn't make it through as you'd expect, despite it feeling closer to the command you invoked.

1. If we let `.env` win and it's not what the user wants, then the workaround is to set variables manually after invoking Fresh.

2. If we let live env vars win and it's not what the user wants, then the workaround is to either not set stuff from `.bashrc` or to somehow read `.env` before invoking Fresh.

I think intuitively I think of the stack as `.bashrc` (system-wide), `.env` (directory default), and then any `export`s the user did before running the command. Unfortunately, the first and last are indistinguishable so we loose either way.

Does any of this help you think that one of the two approaches is better? I'm neutral/unsure at the moment.

Reply:
Hmm - is there maybe a way to have separate options for "forward environment variables" and "use variables in `.env`"? i.e., always make the behavior explicit?

Reply:
Separate option is probably worth pursuing but would require more effort than I have mental space for right now :/ 

I reworked it so that `.env` wins. If you have set `MW_|MEDIAWIKI_` in your `.bashrc` or whatever, because you use a variety of environments for example, you'd want the `.env` version to win when using mediawiki-docker so that you end up with the correct values.